dataprofiler/labelers/column_name_model.py

```diff
@@ -186,9 +181,6 @@ def predict(
         batch_size=None,
         show_confidences=False,
         verbose=True,
-        include_label=True,
-        negative_threshold_config=50,
-        positive_threshold_config=85,
```

Comment:
parameters to the class __init__ not the predict method. Keeping the API consistent across all the models